pkg/combustion/kubernetes.go

```diff
@@ -114,16 +121,23 @@ func configureRKE2(ctx *image.Context) (string, error) {
 		return "", fmt.Errorf("downloading RKE2 artefacts: %w", err)
 	}
 
+	manifestsPath := ""
+	if isComponentConfigured(ctx, filepath.Join(k8sDir, "manifests")) {
```

Comment:
This is not really correct since it doesn't include manifests from the image definition.